### ISSUE TITLE
Accessibility: Component is not keyboard navigable and missing labels

### ISSUE THREAD

**Issue by thedamon** [XiongAmao/vue-easy-lightbox#59]
Currently this component is not navigable by keyboard. The simplest way to fix this is to make any clickable control a button element (including the images that trigger the lightbox, though that might be a change in documentation more than code).

All the icon toolbars also need a text description (aria-label) describing what it does. In order to make it fully translateable you might want to expose a way to configure the messages (possibly via a vue-i18n)

**Comment by NickHatBoecker** [comment]
Also currently you cannot add alttext to images :(
